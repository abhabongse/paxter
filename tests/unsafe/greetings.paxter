@python##"
    import itertools

    def strip(text):
        return flatten(text).strip()

    _symbols_ = {
        '@': '@',
    }
    name = "John Smith"
    age = 24
    counter = itertools.count(start=1)
"##\
My name is @strip{ @name } and I am currently @age years old.
My email is @"john@example.com" or smith@@example.com.
@python"age_next_year = age + 1"\
My age next year would be @age_next_year.

Do you know that 1 + 1 = @|1 + 1|?
@|next(counter)|. Item one.
@|next(counter)|. Item two.
@|next(counter)|. Item three.
