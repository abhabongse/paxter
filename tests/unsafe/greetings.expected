My name is John Smith and I am currently 24 years old.
My email is john@example.com or smith@example.com.
My age next year would be 25.

Do you know that 1 + 1 = 2?
1. Item one.
2. Item two.
3. Item three.
